src/test/scala/com/cognite/sdk/scala/v1/DataModelInstancesTest.scala

```diff
@@ -173,9 +173,8 @@ class DataModelInstancesTest
       )
       .unsafeRunSync()
 
-    // TODO change this when ingest api return better error detail
-    exception.message.contains("Internal server error. Please report this error to") shouldBe true
-
+    exception.message.contains("invalid input") shouldBe true
+    exception.message.contains("abc") shouldBe true
```

Comment:
ah so ingest api returns explicit error now hehe 😁 👍

Reply:
Yeah, full message is `invalid input syntax for type double precision: "abc"` but keeping the check a bit less specific on purpose.